prometheus-ksonnet/mixins.libsonnet

```
@@ -0,0 +1,44 @@
+{
+  // Add you mixins here.
+  mixins+:: {
+    kubernetes+:
```

```suggestion
    kubernetes:
```

Don't think we need to merge here. This could cause hard to debug issues in the future